src/components/test/Checkbox-test.js

```diff
@@ -0,0 +1,106 @@
+import { mount } from 'enzyme';
+
+import { checkAccessibility } from '../../../test/util/accessibility';
+
+import { LabeledCheckbox, Checkbox } from '../Checkbox';
+
+describe('Checkbox', () => {
+  function createComponent(props = {}) {
+    return mount(<Checkbox name="my-checkbox" {...props} />);
+  }
+
+  it('inputRef points to the input element', () => {
```

Comment:
The wording of this test description is a tad awkward.

Reply:
Changed to "passes along a `ref` to the input element through `inputRef`" (copied from the button component)